src/pages/documentation/mobile-core/configuration/api-reference.md

```diff
@@ -62,7 +56,7 @@ Android
 
 ## configureWithFileInPath
 
-You can include a bundled JSON configuration file in your app package to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](./index.md#configure-with-app-id-per-environment) approach.
+You can include a bundled JSON configuration file in your app package to replace or complement the configuration that was downloaded by using the [Configure with App ID per environment](#configure-with-app-id-per-environment) approach.
```

Comment:
I don't know if this link update is correct? Isn't the "Configure with App ID per environment" listed on the overview page? Not in the API?

Reply:
Whoops. Yes. You are right. This should be `.#configure-with-app-id-per-environment`. I incorrectly used the version from Overview.md